internal/dependencies/tools/validator.go

```diff
@@ -37,17 +37,19 @@ func (tv *Validator) Validate(kfdManifest config.KFD) []error {
 
 	tls := reflect.ValueOf(kfdManifest.Tools)
 	for i := 0; i < tls.NumField(); i++ {
-		name := strings.ToLower(tls.Type().Field(i).Name)
+		for j := 0; j < tls.Field(i).NumField(); j++ {
+			name := strings.ToLower(tls.Field(i).Type().Field(j).Name)
```

Comment:
this can be moved down below before `tool := tv.toolFactory.Create(name, version.String())` and save some computation in case the two ifs below go wrong